src/components/expansion/ExpansionPanel.svelte

```diff
@@ -74,6 +73,7 @@
 
   $: createButtonEnabled = seqIdInput !== '';
   $: expandButtonEnabled = selectedExpansionSetId !== null;
+  $: console.log($filteredExpansionSequences);
```

Comment:
Remove log

Reply:
oh i thought this was an essential feature...